.storybook/main.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  "stories": [
+    "../stories/**/*.stories.mdx",
+    "../stories/**/*.stories.@(js|jsx|ts|tsx)"
+  ],
+  "addons": [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/preset-scss",
+    "storybook-addon-mock/register"
+  ],
+  "framework": "@storybook/react"
+}
```

Comment:
**disclaimer**: I might steal this storybook configuration for the whole SDK if it works as described. I tried to get it working the other day but couldn't.

Reply:
`npm run storybook` It made development of the component styles so much easier.